project/Build.scala

```diff
@@ -1147,6 +1147,13 @@ object Build {
            |  > git submodule update --init
         """.stripMargin)
     }
+
+    val vscodeSetting = new File(".vscode/settings.json")
+    if(!vscodeSetting.exists()) {
+      val vscodeSettingTemplate = new File(".vscode/settings-template.json")
```

Comment:
I would use a .vscode-template directory instead:
- We might want to add settings for other files in .vscode
- This means that it's safe to gitignore everything in .vscode